share/spice/coupons/coupons.js

```diff
@@ -0,0 +1,88 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_coupons = function(api_result){
+        // Validate the response (customize for your Spice)
+        if (!api_result || api_result.error) {
+            return Spice.failed('coupons');
+        }
+
+        var tabname, translation_ValidUntil, translation_Unknown;
+        switch (api_result.locale) {
+            case 'DE':
+                tabname = 'Gutscheine';
```

Comment:
Removed the tab name so it's set to "Answers".
You're right, this should not confuse anyone, as I expect our IA to be very seldom be displayed along with other IAs using a generic name.